force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +469,13 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
-    }
-}
\ No newline at end of file
+        if (person1.getAge() > person2.getAge()) {
+            return person1;
```

Comment:
Try to combine this if statement with the else since they are both returning the same person